Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM python:3.7-slim-stretch
 
 RUN pip install pipenv==2018.11.26 \
   && pip install awscli==1.11.174
-RUN apt update && apt install -y libsnappy-dev build-essential
+RUN apt update && apt install -y libsnappy-dev build-essential libpq-dev
```

Comment:
Needed for `psycopg2`. 
http://initd.org/psycopg/docs/install.html#change-in-binary-packages-between-psycopg-2-7-and-2-8